### ISSUE TITLE
Rockets and Rocket launchers

### ISSUE THREAD

**Issue by LordB4** [GiantLuigi4/Omni-Shot#2]
Please try using rockets on guns and bows and also try using arows and bullets with rocket launcher. You will see some things that are not even in game. (Maybe problem with item IDs i am not devepoper of anything)

It is my favorite mod anyway.

**Comment by GiantLuigi4** [comment]
Yeah, I’m well aware of this issue, but I don’t think it’s possible to fox it without manually mapping every single id, which then makes it incompatible with every mod if I try to map modded projectiles.

I’m not even going to really attempt to fix this issue.

(Sorry for late relpy, I haven’t played terraria in a long time)

Edit: Gonna leave this open though in case someone knows how to fix it and wants to pr it.
